modules/cvv/CMakeLists.txt

```diff
@@ -15,8 +15,14 @@ endif()
 
 find_package(Qt${QT_VERSION_MAJOR} REQUIRED COMPONENTS ${CVV_QT_MODULES})
 
-set(CMAKE_AUTOMOC ON)
-set(CMAKE_INCLUDE_CURRENT_DIR ON)
```

Comment:
It does not work. See https://github.com/opencv/opencv/blob/eab21b61067385515c77783608f8c374bfe131c6/modules/highgui/CMakeLists.txt#L91